ptf/tests/unary/slicing.py

```diff
@@ -265,6 +265,113 @@ def runTest(self):
                                     use_default_tc,
                                 )
 
+@group("upf")
+class FabricUpfUplinkWithMeterTest(UpfSimpleTest, SlicingTest):
+    """Tests meters for UPF. This is mostly a dummmy test class to verify
+    basic programming of UPF meters. QoS test for UPF meters and color-aware
+    meter behaviour should use linerate traffic generation. """
+
+    @tvsetup
+    @autocleanup
+    def doRunTest(
+        self,
+        pkt,
+        app_bitrate,
+        session_bitrate
+    ):
+        upf_slice_id = 11
+        upf_tc = 2
+        eg_port = self.port2
+
+        self.runUplinkTest(
+            ue_out_pkt=pkt,
+            tagged1=True,
+            tagged2=True,
+            with_psc=False,
+            is_next_hop_spine=False,
+            slice_id=upf_slice_id,
+            tc=upf_tc,
+            eg_port=eg_port,
+            app_max_bitrate=app_bitrate,
+            session_max_bitrate=session_bitrate,
```

Comment:
Avoids any ambiguity on the units (applies to other instances of `_bitrate`)
```suggestion
            app_max_bps=app_bps,
            session_max_bps=session_bps,
```